hera_cal/abscal.py

```diff
@@ -2845,6 +2846,9 @@ def post_redcal_abscal(model, data, flags, rc_flags, min_bl_cut=None, max_bl_cut
         phs_conv_crit: convergence criterion for updates to iterative phase calibration that compares
             the updates to all 1.0s.
         refant_num: integer antenna number defined to have 0 phase. If None, refant will be automatically chosen.
+        use_idealized_antpos : bool, Instead of actual antenna positions, use redcal to generate idealized
```

Comment:
If this is false, then you're not correcting exactly the same degeneracies that redcal is removing. This was actually an issue that came up in validation step 2.0. I explain the issue in #451 and #453. I don't think we want this option in an explicitly post-redcal abscal. 

However, I can imagine wanting an option to turn the solved phase/delay slopes into physical units... we can chat when you get back about how best to do that.